ListApp/ListApp.Api/Utils/ServerInit.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Http;
+using System.Web.Http.Routing;
+using JsonPatch;
+using JsonPatch.Formatting;
+using JsonPatch.Paths.Resolvers;
+using ListApp.Api.Filters;
+using Microsoft.Web.Http.Routing;
+
+namespace ListApp.Api.Utils
+{
+    public static class ServerInit
+    {
+        public static HttpConfiguration InitializeConfiguration(HttpConfiguration config)
```

Comment:
This should be written in style of `WebApiConfig.cs` and located in App_Start folder. = Register method that would be used in Global.asax

Reply:
Oh...you are using in tests...Is it really necessary? Could you solve the duplicate code that this contains (with WebApiConfig)?